ui.html

```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
```

Comment:
Too much components)

Reply:
I agree, but we had task to write all typography  elements)
So what I should to do? (remove some elements).